csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5187,6 +5187,8 @@ CSAF: https://domain.tld/security/csaf/provider-metadata.json
 CSAF: https://www.example.com/.well-known/csaf/provider-metadata.json
 ```
 
+> An issuing party MAY advertise more than one `provider-metadata.json` by adding multiple `CSAF` fields, e.g. in case of changes to the organisational structure through merges or aquisitions. It is strongly recommended NOT to do this. If it is inevitable, this SHOULD be a temporary solution. If one of the URLs fulfills requirement 9, this SHOULD be used as the first CSAF entry.
```

Comment:
Nit: `organisational` &rightarrow; `organizational`? Should be in American English spelling but it is to late in the day for me to be sure …